app/services/proofing/lexis_nexis/ddp/proofer.rb

```diff
@@ -15,10 +15,9 @@ class Proofer < LexisNexis::Proofer
                             :city,
                             :state,
                             :zipcode,
-                            :request_ip,
-                            :uuid_prefix
+                            :request_ip
 
-        optional_attributes :address2, :phone, :email
+        optional_attributes :address2, :phone, :email, :uuid_prefix
```

Comment:
this is technically not the same change as logging, but I figured I'd update it while I'm here